pipes/WDL/tasks/tasks_nextstrain.wdl

```diff
@@ -1373,6 +1373,7 @@ task refine_augur_tree {
         File     msa_or_vcf
         File     metadata
 
+        Boolean? generate_timetree = true
```

Comment:
```suggestion
        Boolean  generate_timetree = true
```

In this case the parameter is required, but has a default, but while that means the user doesn't need to set anything that doesn't mean it's `?` optional.

Reply:
Makes sense, thank you so much!